SeaPublicWebsite/Views/EnergyEfficiency/OutdoorSpace.cshtml

```
@@ -34,6 +35,17 @@
                             air to water that is sent to your radiators or underfloor heating, as well as a hot water cylinder for providing hot water.
                         </p>
                      </text>;
+                Func<object, object> blueBannerHint = 
+                    @<p class="govuk-body">
+                        Homes of your type typically
+                        <strong>
+                            @(Model.PropertyType switch {
+                                PropertyType.House or PropertyType.Bungalow => "have some outdoor space",
+                                PropertyType.ApartmentFlatOrMaisonette => "do not have any outdoor space"
+                                })
```

Maybe throw an exception for the default case